### PR TITLE
Add featured artists api and form

### PR DESCRIPTION
**Pull request by MauricioDinki**

### CODE REVIEW

**Comment by Enrique199413** [comment]
Consider use Geb (find similar for Python) for functionality test.

**Comment by MauricioDinki** [comment]
@jresendiz27  The function now have a test

**Comment by jresendiz27** [comment]
@MauricioDinki needs to be linked with Jenkins, because I need to see the tests passing

File: latatuadora/apps/api/tests.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+
+    def test_api_can_get_artists(self):
+        """Test the api can get all artists."""
+        # Get all the artists
+        response = self.client.get(
+            reverse('api_featured'),
+            format="json"
+        )
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
```
Comment (jresendiz27) [inline latatuadora/apps/api/tests.py]: Why are you making an assertion with the response? that does not mean that actually follows a Rest convention. Test all the codes needed.

By the way, also consider testing that the response actually contains information  (the one you're inserting or using).

**Pull request closed without merging**